geode-core/src/main/java/org/apache/geode/internal/cache/tier/sockets/Message.java

```diff
@@ -134,7 +134,8 @@
   /**
    * maximum size of an outgoing message. See GEODE-478
```

Comment:
Let's remove the reference to the ticket and include any important information from that ticket here in the comment. I doubt there is anything applicable to include though.

Reply:
👍